templates/BreadcrumbsTemplate.ss

```diff
@@ -1,6 +1,10 @@
-<% if $Pages %>
-    <nav class="breadcrumbs" aria-label="Breadcrumbs">
-        <ol class="breadcrumbs__list">
+<nav class="breadcrumbs" aria-label="Breadcrumbs">
+    <ol class="breadcrumbs__list">
+        <li class="breadcrumbs__list-item">
+            <a href="$BaseURL">Home</a>
+            <span class="breadcrumbs__list-delimiter" aria-hidden="true">/</span>
+        </li>
+        <% if $Pages %>
```

Comment:
```suggestion
```
We technically don't need this (if there's nothing in the list, it doesn't get iterated so nothing in the loop will be rendered) - but if this doesn't change I'll still merge.

Reply:
I'm a bit trigger happy with the gating of loops due to some bad experiences in the past, will remove